src/components/App.js

```diff
@@ -23,7 +25,22 @@ const App = () => {
           <Route exact path="/" element={<Destinations />} />
           <Route path="/register" element={<Register />} />
           <Route path="/login" element={<Login />} />
-          <Route path="/home" element={<Home />} />
+          <Route
+            path="/home"
+            element={(
+              <PrivateRoute redirectTo="/login">
+                <Home />
+              </PrivateRoute>
+        )}
+          />
+          <Route
+            path="/reserve"
+            element={(
+              <PrivateRoute redirectTo="/login">
+                <Reservation />
+              </PrivateRoute>
+        )}
+          />
```

Comment:
Why instead this combination of Route and PrivateRoute components you don't use just Route? Something like this: 
```
<Route path="/home" element={<Home />} />
<Route path="/reserve" element={<Reservation />} />
```
In my case this solved the error I was getting earlier.

Reply:
thanks for pointing it out @ShinobiWarior. I will fix it as you suggest.